README.md

```diff
@@ -181,7 +181,7 @@ tasks.named("dependencyUpdates").configure {
 ```kotlin
 import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 
-tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
+tasks.withType<DependencyUpdatesTask> {
```

Comment:
I recently learned withType isn't enough to be lazy it seems

https://github.com/gradle/gradle/pull/15979#discussion_r612980530

Reply:
I take a look.